metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures ms_dotnet'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '4.2.1'
+version          '4.2.2'
```

Comment:
Please do n ot change the metadata we'll do the proper release once your request has been merged

Reply:
Sure, sorry about that, reverted.